.circleci/config.yml

```diff
@@ -159,13 +159,20 @@ jobs:
       - attach_workspace:
           # Must be absolute path or relative path from working_directory
           at: .
+      # https://superuser.com/a/1620000/54442
+      - run:
+          name: Install and set Node 16.x
+          command: |
+            $ErrorActionPreference="Stop"
+            nvm arch
+            nvm install 16 all
+            nvm use 16
```

Comment:
Where is nvm obtained? Does CircleCI have it by default?

Reply:
Yes, the image has https://github.com/coreybutler/nvm-windows installed

Reply:
Do you know if that dependency is something we configured?